src/Application.php

```diff
@@ -105,4 +100,32 @@ public function run()
 
         return $this;
```

Comment:
As noted in weierophinney/zf-apigility#5, I mistook this as being in the exception handler. On review, this is in `run()`, and your suggestion would replace two lines with a single call.